src/main/java/com/greeenai/greeenai/global/util/MemberUtil.java

```diff
@@ -15,8 +17,8 @@ public class MemberUtil {
 
     // 현재 로그인한 Member를 조회합니다.
     public Member getCurrentMember() {
-        // todo: throw CustomException
-        return memberRepository.findById(getCurrentMemberId()).orElseThrow(() -> new IllegalArgumentException());
+        return memberRepository.findById(getCurrentMemberId())
+                .orElseThrow(() -> new CustomException(ErrorCode.UNAUTHORIZED));
```

Comment:
ErrorCode를 static import하고 아래와 같이 정리하면 좋을 것 같습니다.
```suggestion
                .orElseThrow(() -> new CustomException(UNAUTHORIZED));
```

Reply:
수정했습니다!